packages/shared/routes/setup/Network.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { appRouter } from '@core/router'
+    import { Button, OnboardingLayout, Text } from 'shared/components'
+    import { newProfile, updateActiveProfile } from '@core/profile'
+    import { mobile } from '@lib/app'
+    import { INetwork, NETWORK, NetworkProtocol, NetworkType } from '@core/network'
+    import { TextType } from 'shared/components/Text.svelte'
+
+    const isPrivateNet = (network: INetwork) => network.type === NetworkType.PrivateNet
+
+    const networks: INetwork[] = Object.values(NETWORK).flatMap(Object.values)
+    const filteredNetworks = $newProfile?.isDeveloperProfile
+        ? [...networks.filter((network) => !isPrivateNet(network)), networks.find(isPrivateNet)] // gets only one private net in array
+        : networks.filter((network) => network.type === NetworkType.Mainnet)
+
+    function getNetworkColor(network: INetwork) {
+        if (isPrivateNet(network)) {
+            return 'gray-500'
+        }
+        return network.protocol === NetworkProtocol.Shimmer ? 'shimmer-highlight' : 'black'
```

Comment:
If we add an iota object to the tailwind config we should be able to just use the protocol as the prefix for the class. i.e.
```suggestion
        return `${network.protocol}-highlight`
```